src/lib/Libutil/misc_utils.c

```diff
@@ -1092,7 +1092,13 @@ break_delimited_str(char *strlist, char delim)
 			}
 
 			arr[i] = strdup(tok);
-			tok = strtok_r(NULL, sep, &saveptr);
+			if (arr[i] == NULL) {
+				pbs_errno = PBSE_SYSTEM;
+				free(list);
+				free_string_array(arr);
+				return NULL;
+			}
+			tok = strtok_r(NULL, ",", &saveptr);
```

Comment:
Good catch.  git rebase screwed up.  break_delimited_str() was break_comma_str() when I started.  It was since changed to a more generic break_delimited_str().  Now break_comma_str() calls break_delimited_str() which looks a whole lot like break_comma_str() did before.  I guess git mistook it and added my code in.  I'll fix this.